addons/website_sale/models/product_product.py

```diff
@@ -160,6 +159,20 @@ def _to_markup_data(self, website):
         """
         self.ensure_one()
 
+        product_price = request.pricelist._get_product_price(
+            self, quantity=1, target_currency=website.currency_id,
+        )
+        product_taxes = self.sudo().taxes_id._filter_taxes_by_company(self.env.company)
+        taxes = request.fiscal_position._value.map_tax(product_taxes)
```

Comment:
Could you please comment on why we need `sudo`?
```suggestion
        product_taxes_sudo = self.sudo().taxes_id._filter_taxes_by_company(self.env.company)
        taxes = request.fiscal_position._value.map_tax(product_taxes_sudo)
```